packages/staking-contracts/src/rewards-allocator.ts

```diff
@@ -18,7 +18,7 @@ export class NoopRewardsAllocator implements RewardsAllocator {
     startTimestamp: number,
   ): Promise<AllocationResult> {
     return {
-      transactionHash: "0x0000000000000000000000000000000000000000",
```

Comment:
The `NoopRewardsAllocator` returns `null` now and not the zero hash. That means when we allocate using `NoopRewardsAllocator` tx will be empty